lib/elixir/test/elixir/kernel/docs_test.exs

```diff
@@ -70,17 +70,21 @@ defmodule Kernel.DocsTest do
       end
     end
 
-    assert_raise ArgumentError, ~r/should be a string, false, keyword list, or nil/, fn ->
-      defmodule AtSyntaxDocAttributesFormat do
-        @moduledoc :not_a_binary
-      end
-    end
-
-    assert_raise ArgumentError, ~r/should be a string, false, keyword list, or nil/, fn ->
-      defmodule AtSyntaxDocAttributesFormat do
-        @moduledoc true
-      end
-    end
+    assert_raise ArgumentError,
+                 ~r/should be either false, nil, a string, or a keyword list/,
+                 fn ->
+                   defmodule AtSyntaxDocAttributesFormat do
+                     @moduledoc :not_a_binary
+                   end
+                 end
+
+    assert_raise ArgumentError,
+                 ~r/should be either false, nil, a string, or a keyword list/,
+                 fn ->
+                   defmodule AtSyntaxDocAttributesFormat do
+                     @moduledoc true
+                   end
+                 end
```

Comment:
Nitpick but could you format these so that the fn -> is at the end of the line and the body is on a new line? That's how we tend to do it. On mobile so hard to be clear but hopefully you got it 😂

Reply:
the formatter will put it back to where it was.

Reply:
Right sorry we usually extract the error message into a variable called "message" and you should be fine!

Reply:
done